pkg/koordlet/runtimehooks/nri/server.go

```diff
@@ -47,8 +47,23 @@ type Options struct {
 	Executor      resourceexecutor.ResourceUpdateExecutor
 }
 
+func (o Options) Validate() error {
+	// a fast check for the NRI support status
+	var completeNriSocketPath string
+	if len(o.NriSocketPath) > 0 {
+		completeNriSocketPath = filepath.Join(system.Conf.VarRunRootDir, o.NriSocketPath)
+	} else {
+		completeNriSocketPath = filepath.Join(system.Conf.VarRunRootDir, "nri/nri.sock")
```

Comment:
it seems this line will never be executed. o.NriSocketPath will never be empty because when we NewDefaultConfig, it will always set RuntimeHooksNRISocketPath to nri/nri.sock. see this https://github.com/koordinator-sh/koordinator/blob/main/pkg/koordlet/runtimehooks/config.go#L102.

Reply:
Yes, it was only for testing. Let me remove it.